nntrainer/models/neuralnet.cpp

```diff
@@ -816,7 +816,7 @@ sharedConstTensors NeuralNetwork::incremental_inference(
 std::vector<float *> NeuralNetwork::incremental_inference(
   unsigned int batch_size, const std::vector<float *> &input,
   const std::vector<float *> &label, unsigned int init_seq_len,
-  unsigned int from, unsigned int to) {
+  unsigned int from, unsigned int to, bool return_last_output_only) {
```

Comment:
How about setting it to "need_all_output" or "need_only_last" instead of "return_last_output_only"?
(I think the word 'return' is unnecessary and could be replaced with a better name)
It seems like an internal flag that determines the return value.

Reply:
I think the current parameter name is also good.
However, if there's a necessity for renaming it, I would recommend considering 'output_hidden_states'. Because this term has been utilized in HuggingFace for similar purposes. (if "output_hidden_states == false", then only return the last output values)

Reply:
If there is a similar case, it would be good to use the same name. 
It seems that developers will easily understand this.

Reply:
Renamed it as @baek2sm suggested.